F/finufft/build_tarballs.jl

```diff
@@ -27,13 +24,19 @@ cmake .. \
     -DCMAKE_INSTALL_PREFIX="${prefix}" \
     -DCMAKE_TOOLCHAIN_FILE="${CMAKE_TARGET_TOOLCHAIN}" \
     -DFINUFFT_FFTW_SUFFIX="" \
-    -DFINUFFT_ARCH_FLAGS=""
+    -DFINUFFT_ARCH_FLAGS="" \
+    -DFINUFFT_STATIC_LINKING="OFF"
```

Comment:
They are supposedly inserted later when the library is built for different microarchitectures, but I don't remember the mechanics of it at the moment

Reply:
Update: They are enforced by scripts that wrap the compiler
https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.generate_compiler_wrappers!-Tuple{AbstractPlatform}

Reply:
Interesting, thanks!

Reply:
if we use recent gcc/clang instead of `-march=avx2` migth be worth trying `-march=x86_64_v2` source:https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html this enables other optimizations too that might be beneficial. Only possible if we can account for the compiler used as it requires recent gcc